elasticdl/python/master/master.py

```diff
@@ -97,6 +98,10 @@ def __init__(self, args):
             args.num_epochs,
         )
 
+        saved_model_path = args.output
```

Comment:
Maybe `args.export_dir` better ?

The args of `tf.saved_model.save` is
```
tf.saved_model.save(
    obj,
    export_dir,
    signatures=None,
    options=None
)

Reply:
Good idea, I prefer rename it in the next PR. The 'output' command name already exist in the document, we need update it in all the documents.